README.md

```diff
@@ -153,93 +186,92 @@ This can be used to represent all kinds of errors. It can be for errors that hap
 | <a name="error.code"></a>`error.code`  | Error code describing the error.  | keyword  |   |   |
 
 
-## <a name="event"></a> Event fields
+### <a name="event"></a> Event fields
 
-The event fields are used for context information about the data itself.
+The event fields are carry context information about the data itself.
 
 
 | Field  | Description  | Type  | Multi Field  | Example  |
 |---|---|---|---|---|
 | <a name="event.id"></a>`event.id`  | Unique ID to describe the event.  | keyword  |   | `8a4f500d`  |
-| <a name="event.category"></a>`event.category`  | Event category.<br/>This can be a user defined category.  | keyword  |   | `metrics`  |
-| <a name="event.type"></a>`event.type`  | A type given to this kind of event which can be used for grouping.<br/>This is normally defined by the user.  | keyword  |   | `nginx-stats-metrics`  |
-| <a name="event.module"></a>`event.module`  | Name of the module this data is coming from.<br/>This information is coming from the modules used in Beats or Logstash.  | keyword  |   | `mysql`  |
-| <a name="event.dataset"></a>`event.dataset`  | Name of the dataset.<br/>The concept of a `dataset` (fileset / metricset) is used in Beats as a subset of modules. It contains the information which is currently stored in metricset.name and metricset.module or fileset.name.  | keyword  |   | `stats`  |
-| <a name="event.severity"></a>`event.severity`  | Severity describes the severity of the event. What the different severity values mean can very different between use cases. It's up to the implementer to make sure severities are consistent across events.  | long  |   | `7`  |
-| <a name="event.raw"></a>`event.raw`  | Raw text message of entire event to be used to demonstrate log integrity.  | keyword  |   | `Sep 19 08:26:10 host CEF:0&#124;Security&#124; threatmanager&#124;1.0&#124;100&#124; worm successfully stopped&#124;10&#124;src=10.0.0.1 dst=2.1.2.2spt=1232`  |
+| <a name="event.category"></a>`event.category`  | Event category.<br/>TBD. This can be a user defined category.  | keyword  |   | `metrics`  |
+| <a name="event.type"></a>`event.type`  | A type given to this kind of event which can be used for grouping.<br/>TBD. This is normally defined by the user.  | keyword  |   | `nginx-stats-metrics`  |
+| <a name="event.module"></a>`event.module`  | Name of the Beats or Logstash module that is the source of the data.  | keyword  |   | `mysql`  |
```

Comment:
I would not limit this to `Beats` or `Logstash` here but rather use the two as examples.